models/app.model.js

```diff
@@ -29,4 +32,16 @@ exports.selectArticles = () => {
                 comment_count: Number(article.comment_count),
             }));
         });
+};
+
+//TASK 6
+exports.selectComments = (article_id) => {
+    const text = `SELECT * FROM comments WHERE article_id = $1;`
+    const values = [article_id];
+    return db.query(text, values).then(({ rows }) => {
+        if (rows.length === 0) {
+            return Promise.reject({ status: 404, msg: "No comments found"});
+        }
```

Comment:
Please do ensure any code you write has been tested for, how would you test that this behaves as you would expect it to?